dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -16,19 +16,21 @@
 
 package io.dockstore.webservice.jdbi;
 
-import java.lang.reflect.ParameterizedType;
```

Comment:
There are a lot of superfluous import movements. 
Check that you've properly imported the code style document https://github.com/ga4gh/dockstore/blob/develop/codestyle.xml#L7